pulsar-client-cpp/docker-build.sh

```diff
@@ -26,7 +26,7 @@ set -e
 ROOT_DIR=$(git rev-parse --show-toplevel)
 cd $ROOT_DIR/pulsar-client-cpp
 
-BUILD_IMAGE_NAME="${BUILD_IMAGE_NAME:-apachepulsar/pulsar-build}"
+BUILD_IMAGE_NAME="${BUILD_IMAGE_NAME:-bewaremypower/pulsar-build}"
```

Comment:
please revert this part.

Reply:
I mean, after we see that CI passed

Reply:
Yeah, after the CI passed. @zymap will push the latest image to `apachepulsar`'s dockerhub and I'll use the latest image to test again.